storage/src/store/mod.rs

```diff
@@ -729,7 +729,9 @@ where
         self.locations
             .prune(self.oldest_retained_loc)
             .await
-            .map_err(Error::Journal)
+            .map_err(Error::Journal)?;
```

Comment:
nit: I suspect you can still return this directly?

Reply:
the result here is now bool, so not until the full PR which also changes the return type of this function to bool